definitions/features/foreman_database.rb

```diff
@@ -16,6 +16,6 @@ def query_csv(sql)
   end
 
   def psql(query)
-    execute("su - postgres -c 'psql -d foreman'", :stdin => query)
+    execute("su - postgres -s '/bin/sh' -c 'cd ~; psql -d foreman'", :stdin => query)
```

Comment:
@iNecas,

Without login i.e `-` or `-l`, it is anyway not going to show errors/warnings from files.
In case of login, it loads `profile files` or `bashrc file` again.

Using `su - postgres -s '/bin/sh'` , it doesn't show any warning messages but yes, it is definitely show syntax or other errors. I thought that other errors including syntax should be reported.
In that case it will again fail.

If we don't need login at all then we can go for `su postgres -c 'cd ~; psql -d foreman'`.
Please confirm. I will make changes accordingly.

Reply:
Lets try and see: if it works, lets use it

Reply:
Done!